### PR TITLE
Add Valheim to supported game servers

### PR DESCRIPTION
**Pull request by RedSparr0w**
Added Valheim to list of supported game servers

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -80,7 +81,7 @@ Here's a rough (and potentially incorrect) list of default ports for the various
 | `ARK`       | UDP 27015 & UDP 7777 & UDP 7778 ([Guide](https://ark.gamepedia.com/Dedicated_Server_Setup))                                                                                                    |
 | `Arma3`     | UDP 2302 to UDP 2306 ([Guide](https://community.bistudio.com/wiki/Arma_3_Dedicated_Server))                                                                                                    |
 | `Factorio`  | UDP 34197 ([Guide](https://wiki.factorio.com/Multiplayer))                                                                                                                                     |
-| `FiveM`     | UDP 30120 & TCP 30120 ([Guide](https://docs.fivem.net/docs/server-manual/setting-up-a-server/))                                                                                                 |
+| `FiveM`     | UDP 30120 & TCP 30120 ([Guide](https://docs.fivem.net/docs/server-manual/setting-up-a-server/))                                                                                                |
```
Comment (RedSparr0w) [inline README.md:84]: Just removed a space to fix alignment.

File: README.md [hunk 2]
```diff
@@ -89,6 +90,7 @@ Here's a rough (and potentially incorrect) list of default ports for the various
 | `SevenDays` | UDP 26900 to UDP 26902 & TCP 26900 ([Guide](https://developer.valvesoftware.com/wiki/7_Days_to_Die_Dedicated_Server))                                                                          |
 | `srcds`     | Depends on the game. Usually UDP 27015. ([List of games under srcds](https://github.com/CubeCoders/AMP/wiki/Supported-Applications-Compatibility#applications-running-under-the-srcds-module)) |
 | `StarBound` | TCP 21025 ([Guide](https://starbounder.org/Guide:Setting_Up_Multiplayer))                                                                                                                      |
+| `Valheim`   | UDP 5678 → 5680                                                                                                                                                                                |
```
Comment (RedSparr0w) [inline README.md:93]: Worth noting that these seem to be the default ports AMP uses, but aren't the default Valheim ports.

File: example-configs/valheim/docker-compose.yml [hunk 3]
```diff
@@ -0,0 +1,20 @@
+# This configuration creates one Factorio instance. No ADS instances are created.
```
Comment (trevjonez) [inline example-configs/valheim/docker-compose.yml:1]: `Factorio` -> `Valheim`

Comment (RedSparr0w) [reply example-configs/valheim/docker-compose.yml:1]:
```suggestion
# This configuration creates one Valheim instance. No ADS instances are created.
```

**Comment by RedSparr0w** [comment]
Should be good for review 👍

**Comment by Joly0** [comment]
Can aggree, valheim works and should be added to the supported list

**Review by MitchTalmadge — APPROVED** [review]
Thanks! Sorry it has taken me so long. I just finished college and I am finally having free time in my life.

**Pull request merged**